recipes-core/meta/meta-toolchain.bbappend

```diff
@@ -2,3 +2,10 @@
 # /usr/local/oe-sdk-hardcoded-buildpath, so avoid causing errors for
 # outgoing symlinks that are there by design.
 CHECK_SDK_SYSROOTS = "0"
+
+# Add the package sources needed when building the SDK with BUILD_IMAGES_FROM_FEEDS
+IPK_FEED_URIS:append:task-populate-sdk = "${@oe.utils.conditional('BUILD_IMAGES_FROM_FEEDS', '1', '\
+    NIOE-dummy-sdk##${NILRT_LOCAL_FEED_URI}/sdk-provides-dummy-target \
+    NIOE-nativesdk##${NILRT_LOCAL_FEED_URI}/${SDK_ARCH}-${SDKPKGSUFFIX} \
+    NIOE-dummy-nativesdk##${NILRT_LOCAL_FEED_URI}/sdk-provides-dummy-${SDKPKGSUFFIX} \
```

Comment:
Is there a way that we can make these URIs generic, and add this logic into the OE-core layer with the other BUILD_IMAGES_FROM_FEEDS stuff?

Reply:
The `NILRT_LOCAL_FEED_URI` could be replaced with `DEPLOY_DIR_IPK` or `DEPLOY_DIR_<IMAGE_PKGTYPE>`. The part I'm unable to figure out is if the name portion before the `##` for each feed is required.